src/Form/Type/AddressType.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace Eckinox\AddressBundle\Form\Type;
+
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\Extension\Core\Type\TextType;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+class AddressType extends AbstractType
+{
+	public function buildForm(FormBuilderInterface $builder, array $options): void
+	{
+		$builder
+			->add('name', TextType::class)
+			->add('address', AddressAutocompleteType::class, [
+				'attr' => [
+					'autocomplete' => uniqid('noautocomplete'),
+				],
+				'api' => $options['api'],
+			])
+			->add('city', TextType::class)
+			->add('province', TextType::class)
+			->add('postalCode', TextType::class)
+		;
+	}
+
+	public function configureOptions(OptionsResolver $resolver): void
+	{
+		$resolver->setDefaults([
+			'data_class' => null,
+			'api' => 'addressComplete',
```

Comment:
Au lieu d'être des values en string comme ça, définir et utiliser des constantes serait plus safe!

Reply:
it is now 
![image](https://user-images.githubusercontent.com/84418961/204854587-0bff118d-4ceb-4dd3-9b97-e0e051570664.png)